CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ All notable changes to this project will be documented in this file.
 - Change default value for counter [@john](https://github.com/tri-et/TPTEA/tree/feature/fix-bugs-counter)
 - Rename variable  to camel case in page MenuDetail 
 - Remove register facebook button from login page into register page
+- Change field create gift card function
```

Comment:
Change field create gift card function  -> Adjust createGiftCard, simplify and shorten jwt by moving Gift Card info into DB